sdk/config.go

```diff
@@ -53,9 +59,11 @@ const hsDatahMacKey = "dataObfuscationhMacKey"
 const defaultInstrumentationDisabled = false
 const defaultCollectorInsecure = false
 const defaultCollectorEndpoint = "collector.heliosphere.io:443"
-const defaultCollectorPath = "traces"
+const defaultCollectorPath = "/v1/traces"
+const defaultCollectorMetricsPath = "/v1/metrics"
```

Comment:
are we supporting both? we are good here?

Reply:
Sure